### ISSUE TITLE
Add license

### ISSUE THREAD

**Issue by fpesari** [guysherman/jack-passthrough#2]
Hello,

I wanted to package this for openSUSE but the lack of licensing info means it's proprietary, so I can't upload it. Could you please add licensing information? A LICENSE containing the text of a license of your choice would be enough. Thank you.

**Comment by fpesari** [comment]
Hello,

I wanted to package this for openSUSE but the lack of licensing info means it's proprietary, so I can't upload it. Could you please add licensing information? A LICENSE containing the text of a license of your choice would be enough. Thank you.

**Comment by PowerUser64** [comment]
Agreed, I would appreciate it if a license could be added too. I would like to package this for [nixpkgs](https://github.com/NixOS/nixpkgs/), and I'm not sure what license to mark it as. For now, I'll have to mark it as unfree.

**Comment by PowerUser64** [comment]
@guysherman any word on this?
